projects/agentfi/strategies/dex-balancer.js

```diff
@@ -57,8 +57,8 @@ async function getTvlForDexBalancer(agentAddresses, api) {
             const reserve1BigN = BigNumber(reserve1)
             const token0Balance = lpBalanceBigN.times(reserve0BigN).div(totalBigN)
             const token1Balance = lpBalanceBigN.times(reserve1BigN).div(totalBigN)
-            api.add(RingTokenMappings[address0], token0Balance.toFixed(0))
-            api.add(RingTokenMappings[address1], token1Balance.toFixed(0))
+            api.add(RingTokenMappings[address0.toLowerCase()], token0Balance.toFixed(0))
```

Comment:
this should not make any difference

Reply:
@g1nt0ki it actually does, I discovered today that the was refactor to use the addresses from the helper on the 14th, and since then defilama stopped updating TVL, run the code and there was error. I think because the some chars in token address from the helper is lowercase, but the token address return from the data read is not, thus it could not find a matching token and throws an error

```
const ADDRESSES = require('../../helper/coreAssets.json')
const { balanceFetcher, uniV2Lp, tokenAddress } = require('../ca')
const BigNumber = require("bignumber.js");


// underlying token mapping
const RingTokenMappings = {
    // fwUSDB -> USDB
    [ADDRESSES.blast.fwUSDB]: tokenAddress.USDB,
    // fwWETH -> WETH
    [ADDRESSES.blast.fwWETH]: tokenAddress.WETH,
};
```

Reply:
Anyway thanks for merging, appreciate the quick response